internal/rbd/rbd_util.go

```diff
@@ -993,6 +993,9 @@ func genVolFromVolumeOptions(ctx context.Context, volOptions, credentials map[st
 }
 
 func (rv *rbdVolume) validateImageFeatures(imageFeatures string) error {
+	if strings.TrimSpace(imageFeatures) == "" {
```

Comment:
@mnaser just above this , please feel free to add some source code comment about the scenario we came across. :+1: